### PR TITLE
Add themeable icons for AOSiP

### PR DESCRIPTION
**Pull request by ohayoubaka**
- **System UI:** Advanced power menu icons

- **Settings Icons:** OwlsNest settings icons

- **Advanced settings:** Additional buttons settings icon

- **Display modes:** Display modes settings icon

### CODE REVIEW

**Comment by skittles9823** [comment]
@ohayoubaka Oof

**Comment by abhijeetbohra** [comment]
@ohayoubaka Wow Pat pro

File: app/src/main/assets/overlays/com.android.settings.icons/type2_No_Circles/values/colors.xml [hunk 1]
```diff
@@ -14,4 +14,5 @@
     <color name="homepage_storage_background">#00000000</color>   <!--OG:#ffc14ce6-->
     <color name="homepage_support_background">#00000000</color>   <!--OG:#ff26459c-->
     <color name="homepage_system_background">#00000000</color>   <!--OG:#ff757575-->
+	<color name="homepage_owlsnest_background">#00000000</color>   <!--OG:#ffff6868-->
```
Comment (MSF-Jarvis) [inline app/src/main/assets/overlays/com.android.settings.icons/type2_No_Circles/values/colors.xml]: indent pls

File: app/src/main/assets/overlays/com.android.settings.icons/type2_Colourful_Circles/values/colors.xml [hunk 2]
```diff
@@ -14,4 +14,5 @@
     <color name="homepage_storage_background">#ffc14ce6</color>
     <color name="homepage_support_background">#ff26459c</color>
     <color name="homepage_system_background">#ff757575</color>
+	<color name="homepage_owlsnest_background">#ffff6868</color>
```
Comment (MSF-Jarvis) [inline app/src/main/assets/overlays/com.android.settings.icons/type2_Colourful_Circles/values/colors.xml]: indent pls

File: app/src/main/assets/overlays/com.android.settings.icons/res/values/colors.xml [hunk 3]
```diff
@@ -14,4 +14,14 @@
     <color name="homepage_storage_background">@*android:color/material_deep_teal_500</color>   <!--OG:#ffc14ce6-->
     <color name="homepage_support_background">@*android:color/material_deep_teal_500</color>   <!--OG:#ff26459c-->
     <color name="homepage_system_background">@*android:color/material_deep_teal_500</color>   <!--OG:#ff757575-->
+	<color name="homepage_owlsnest_background">@*android:color/material_deep_teal_500</color>   <!--OG:#ffff6868-->
+	<color name="owlsnest_oval_advanced">#00000000</color>   <!--OG:#ffff6868-->
```
Comment (MSF-Jarvis) [inline app/src/main/assets/overlays/com.android.settings.icons/res/values/colors.xml]: indent pls

**Pull request merged**